docs/application/native/guides/text-input/autofill.md

```diff
@@ -223,17 +206,14 @@ To retrieve the data previously entered by the user and stored in an Autofill se
 
         dlog_print(DLOG_INFO, LOG_TAG, "id : %s, value : %s, presentation text : %s", id, value, presentation_text);
 
-        if (id) {
+        if (id)
```

Comment:
Line 191: @imfine98,  @mijin85cho comment not fixed "for the requested view ID."

Reply:
Fixed.